back/lib/shared/helpers/setStatusAndMessages.go

```diff
@@ -0,0 +1,9 @@
+package helpers
+
+import "github.com/gofiber/fiber/v2"
+
+func SetStatusAndMessages(status int, message string, ctx *fiber.Ctx) error {
```

Comment:
When using generic words like `Status`, please refine them by adding what they are related to?

HTTP status? Order status?

Reply:
I'll make a rework to make it clearer. Thanks!